src/components/tx-flow/flows/SuccessScreen/index.tsx

```diff
@@ -71,8 +71,8 @@ export const SuccessScreen = ({ txId }: { txId: string }) => {
       <Divider />
       <div className={classnames(css.row, css.buttons)}>
         <Link href={homeLink} passHref>
-          <Button variant="outlined" size="small">
-            Back to dashboard
```

Comment:
I'd sooner leave this as is. We lose information changing this.

Reply:
What information? "Back to dashboard" is plain incorrect in most cases. It doesn't go to dashboard, it just closes the modal.

Reply:
>What information? "Back to dashboard" is plain incorrect in most cases. It doesn't go to dashboard, it just closes the modal.

We lose the context of where it navigates to - [the dashboard](https://github.com/safe-global/safe-wallet-web/blob/can-sign/src/components/tx-flow/flows/SuccessScreen/index.tsx#L41-L44).

Reply:
Ah, I see. I'll remove the link and will make it close the modal.